### PR TITLE
Fix write support

### PR DESCRIPTION
**Pull request by angguss**
#29

### CODE REVIEW

File: src/duckx.cpp [hunk 1]
```diff
@@ -152,20 +152,67 @@ void duckx::Document::open() {
...
+	// Loop & copy each relevant entry in the original zip
+	int orig_zip_entry_ct = zip_total_entries(orig_zip);
+	for (int i = 0; i < orig_zip_entry_ct; i++)
+	{
+		void* orig_entry_data;
+		zip_entry_openbyindex(orig_zip, i);
+		const char* name = zip_entry_name(orig_zip);
+		// Skip copying the original file
+		if (std::string(name) != std::string("word/document.xml") 
+			&& std::string(name) != std::string("/word/document.xml"))
```
Comment (ehsan-mohammadi) [inline src/duckx.cpp]: Hello @angguss! Many thanks for your PR. I just have a question. Can you describe to me what did you add this line?

File: src/duckx.hpp [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #define DUCKX_H
 
 #include <stdlib.h>
+#include <cstdio>
```
Comment (ehsan-mohammadi) [inline src/duckx.hpp]:
```suggestion

```

@angguss Also, I think we don't need to add this header file. I removed this header file and compiled DuckX and then I saw everything works perfectly.

Comment (angguss) [reply src/duckx.hpp]: Without this header the build fails in to build in TravisCI - the original PR didn't include it

File: src/duckx.cpp (hunk 1 above)
Comment (angguss) [reply src/duckx.cpp]: my mistake, should not have been left in

File: src/duckx.hpp (hunk 2 above)
Comment (ehsan-mohammadi) [reply src/duckx.hpp]: @angguss Can you describe to me why build fails in TravisCI?

**Pull request merged**